File: app/src/main/java/com/qianlei/jiaowu/entity/Student.java
package com.qianlei.jiaowu.entity;

import com.alibaba.fastjson.annotation.JSONField;

import lombok.Data;

/**
 * 学生信息的实体类
 *
 * @author qianlei
 */
@Data
public class Student {
    @JSONField(name = "xh_id")
    private String studentId;
    @JSONField(name = "xm")
    private String name;
}

File: app/src/main/java/com/qianlei/jiaowu/entity/Student.kt
package com.qianlei.jiaowu.entity

import com.google.gson.annotations.SerializedName

/**
 * 学生信息的实体类
 *
 * @author qianlei
 */
class Student {
    @SerializedName(value = "xh_id")
    var studentId: String? = null

    @SerializedName(value = "xm")
    var name: String? = null
}